.github/workflows/php-package-ci.yml

```
@@ -0,0 +1,23 @@
+name: Package CI
+
+on:
+  pull_request:
+
+jobs:
+  checks:
+    name: Checks
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ 7.1, 7.2, 7.3, 7.4, 8.0 ]
```

```suggestion
        php: [ 7.4, 8.0 ]
```
Buďme jednotní s testovanými verzemi https://github.com/Kdyby/Events/pull/126/files#diff-c21bebe73b3162e2f53479410112c0509a17ac0ad86e5b0fb29ceefd610f80b0